pkg/onit/cli/cli.go

```diff
@@ -1121,20 +1123,66 @@ func getDebugCommand() *cobra.Command {
 			if err != nil {
 				exitError(err)
 			}
-
 			port, _ := cmd.Flags().GetInt("port")
-			if err := cluster.PortForward(args[0], port, 40000); err != nil {
+			nodes, err := cluster.GetNodes()
+			if err != nil {
 				exitError(err)
+			}
+
+			if len(args) == 0 {
+				var wg sync.WaitGroup
+				n := len(nodes)
+				wg.Add(n)
+
+				asyncErrors := make(chan error, n)
+				debugPort := onit.DebugPort
+				checked := make(chan bool, 1)
```

Comment:
It seems like we don't need a channel here. This is just being used in a synchronous check, right? `CheckFreePort` can just return a `bool`.